sdk/keyvault/azure-keyvault-administration/tests/test_access_control_async.py

```diff
@@ -94,7 +94,7 @@ async def test_role_definitions(self, client):
 
         async for d in client.list_role_definitions(scope):
             assert (d.id != definition.id), "the role definition should have been deleted"
-        await asyncio.sleep(30)  # additional waiting to avoid conflicts with resources in other tests
+        await asyncio.sleep(60)  # additional waiting to avoid conflicts with resources in other tests
```

Comment:
Are these tests filtered to only run in live mode so we can avoid sleeps in CI?

Reply:
That's a great point -- I should definitely do that. Thank you for pointing that out!